README_DEVELOPERS.md

```diff
@@ -43,3 +43,22 @@ npm run test -- -- -t react
 ## Replace etalon
 
 To replace etalon image just remove it from `packages/devextreme-cli/testing/__tests__/__image_snapshots__` folder and run tests again
+
+# Updating Templates
+
+It moves the changes from the generated application in `testing/sandbox` to the template in `templates` according to the framework.
```

Comment:
Maybe  you pock some word combination from this:

> After you modified the generated application you can apply these changes to the app-template itself by running the auto-updating script.

Actually, I don't know which is better, it'is a hard thing to write a definition for.